Sources/SKTestSupport/Utils.swift

```diff
@@ -0,0 +1,44 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import LanguageServerProtocol
+
+extension Language {
+  var fileExtension: String {
+    switch self {
+    case .objective_c: return "m"
```

Comment:
`objective_cpp` could probably use a value here as well, though I assume we don't have tests with it?

Reply:
Yeah, I wanted to add a case here once/if we ever have an Objective-C++ test case.